src/commands/importFunctionApp/importFunctionApp.ts

```diff
@@ -29,7 +27,8 @@ export async function importFunctionAppToApi(node?: ApiTreeItem): Promise<void>
     ext.outputChannel.show();
     ext.outputChannel.appendLine(localize("importFunctionApp", `Import Function App started...`));
```

Comment:
too verbose remove

  ext.outputChannel.appendLine(localize("importFunctionApp", `Import Function App started...`));

    ext.outputChannel.appendLine(localize("importFunctionApp", "Getting Function Apps..."));

Reply:
Please use the progress bars and information windows

Reply:
we may rethink about this, since 1. this can be very fast, (within one second), which will make the progress window just suddenly popup and disappear, it may cause confusion, if we indeed want to track this, we may add this functionality into the existing progress window...